engine/job.go

```diff
@@ -235,22 +214,17 @@ func (job *Job) Printf(format string, args ...interface{}) (n int, err error) {
 	return fmt.Fprintf(job.Stdout, format, args...)
 }
 
-func (job *Job) Errorf(format string, args ...interface{}) Status {
+// TODO: still used in daemon/create.go, to be deleted
+func (job *Job) Errorf(format string, args ...interface{}) {
```

Comment:
you can remove this function too, simply use `log.Printf()` make sure it's the logrus package that's renamed to log.